### ISSUE TITLE
AssertionError: Unable to add an error, `column` should be a positive number but -2 given

### ISSUE THREAD

**Issue by feross** [jscs-dev/node-jscs#1006]
A user of [`standard`](https://github.com/feross/standard) just ran into [this issue](https://github.com/jscs-dev/node-jscs/issues/857) again, which was supposedly already fixed.

This is the .jscsrc file: https://github.com/feross/standard/blob/master/rc/.jscsrc

Looks like a `validateIndentation` bug. /cc @mikesherov

```
AssertionError: Unable to add an error, `column` should be a positive number but -2 given
    at Object.Errors.add (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/errors.js:43:9)
    at /Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/rules/validate-indentation.js:187:32
    at Array.forEach (native)
    at checkIndentations (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/rules/validate-indentation.js:180:26)
    at Object.module.exports.check (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/rules/validate-indentation.js:430:9)
    at null.<anonymous> (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/string-checker.js:121:22)
    at Array.forEach (native)
    at StringChecker.checkString (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/string-checker.js:119:35)
    at null.<anonymous> (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/lib/checker.js:40:25)
    at Array.1 (/Users/jakechampion/.nvm/v0.10.28/lib/node_modules/standard/node_modules/jscs/node_modules/vow/lib/vow.js:699:39)
```

From: https://github.com/feross/standard/issues/26#issuecomment-73093081

**Comment by mikesherov** [comment]
@feross, you need to wait until a release happens that contains the fix. 1.11 will be out tomorrow.

**Comment by feross** [comment]
Oh, somehow I thought it was released in 1.10. Thanks!
